src/web_client.h

```diff
@@ -1,22 +1,30 @@
 #ifndef NETDATA_WEB_CLIENT_H
 #define NETDATA_WEB_CLIENT_H 1
 
-#define DEFAULT_DISCONNECT_IDLE_WEB_CLIENTS_AFTER_SECONDS 60
-extern int web_client_timeout;
+/**
+ * @file web_client.h
+ * @brief API of the web client.
+ */
+
+#define DEFAULT_DISCONNECT_IDLE_WEB_CLIENTS_AFTER_SECONDS 60 ///< default timeout to disconnect idle web clients.
+extern int web_client_timeout;                               ///< seconds to disconnect timeouts.
 
 #ifdef NETDATA_WITH_ZLIB
-extern int web_enable_gzip, web_gzip_level, web_gzip_strategy, web_donotrack_comply;
+extern int web_enable_gzip;      ///< enable gzip compressed web traffic
+extern int web_gzip_level;       ///< gzip level to use
+extern int web_gzip_strategy;    ///< gzip strategy to use
+extern int web_donotrack_comply; ///< boolean: Use do not track.
 #endif /* NETDATA_WITH_ZLIB */
 
-#define WEB_CLIENT_MODE_NORMAL      0
-#define WEB_CLIENT_MODE_FILECOPY    1
-#define WEB_CLIENT_MODE_OPTIONS     2
+#define WEB_CLIENT_MODE_NORMAL      0 ///< Standard web client mode.
+#define WEB_CLIENT_MODE_FILECOPY    1 ///< ktsaou: Your help needed.
+#define WEB_CLIENT_MODE_OPTIONS     2 ///< ktsaou: Your help needed.
```

Comment:
HTTP `OPTIONS` request, i.e. don't do the job, just respond if it is allowed. Used to enable cross-domain CORS.